checkstyle-tester/README_MANUAL_EXECUTION.md

```diff
@@ -11,7 +11,9 @@ external files and branches first.
 Anything that starts with `#` is considered a comment and is ignored.
 `projects-to-test-on.properties` is expected to be in the following format:
 
-> REPO_NAME|[local|git|hg]|URL|[COMMIT_ID]|[EXCLUDE FOLDERS]
+```properties
+REPO_NAME|[local|git|hg]|URL|[COMMIT_ID]|[EXCLUDE FOLDERS]|
+```
```

Comment:
Please do
```

`REPO_NAME|[local|git|hg]|URL|[COMMIT_ID]|[EXCLUDE FOLDERS]|`

```

Reply:
i see missing pipes or extra pipes. even if it doesn't lead to any ci issue, but let me know if it is an issue and needs correction. 
```
```properties
my_custom_project|local|/home/username/java/my_custom_project||
my_custom_checkstyle|git|/home/username/java/git-repos/checkstyle/checkstyle|master|**/ignoreFolderOrFiles/**/*
my_custom_repo|hg|/home/username/java/hg-repos/myRepo|default|
```
```

Reply:
It should be single line and naturally wrapped by browser

Reply:
<img width="693" alt="image" src="https://github.com/checkstyle/contribution/assets/95259545/ca5bfca1-eb82-4505-9f34-40a989c75245">
this is how it looks in broswer